liferay-workspace/tests/build.gradle

```diff
@@ -0,0 +1,14 @@
+apply plugin: 'groovy'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testCompile 'org.codehaus.groovy:groovy-all:2.4.4'
+    testCompile 'org.spockframework:spock-core:1.0-groovy-2.4'
+	testCompile gradleTestKit()
+}
+
+check.dependsOn ':blade-tests:initBundle'
+check.mustRunAfter ':liferay-workspace/modules:build'
```

Comment:
Also lets switch it to `test.dependsOn` instead of `check.dependsOn` since `check` is greater than `test`.  We really just need `test`